app/views/dashboard/epic_queue_records/index.json.jbuilder

```diff
@@ -3,7 +3,7 @@ json.rows do
   json.(@epic_queue_records.limit(params[:limit]).offset(params[:offset])) do |eqr|
     json.protocol_id  eqr.protocol_id
     json.protocol     format_protocol(eqr.protocol)
-    json.notes        notes_button(eqr)
+    json.notes        notes_button(eqr, protocol_id: eqr.protocol.id)
     json.pis          format_pis(eqr.protocol)
     json.date         format_epic_queue_created_at(eqr)
     json.status       eqr.status.capitalize
```

Comment:
See comment about combining `send` and `delete` into `actions`, can you make that change here as well. You can add a helper that joins the send and delete buttons (see `authorized_user_actions`, `document_actions`, etc. for examples)